internal/bgp/frr/frr_test.go

```diff
@@ -520,3 +521,35 @@ func TestTwoAdvertisementsTwoSessions(t *testing.T) {
 
 	testCheckConfigFile(t)
 }
+
+func TestLoggingConfiguration(t *testing.T) {
+	testSetup(t)
+
+	l := log.NewNopLogger()
+	sessionManager := NewSessionManager(l, logging.LevelWarn)
+	defer close(sessionManager.reloadConfig)
+
+	config, err := sessionManager.createConfig()
```

Comment:
how about creating a dumb session instead of manually doing this? It would be more aligned to how this is really used (instead of writing manually to the channel)

Reply:
Good point. It would be nearer to the real usage of SessionManager. Anyway, the bare minimum code to get it a session is:

```
	session1, err := sessionManager.NewSession(l, "10.2.2.255:179", net.ParseIP("10.1.1.254"), 0, net.ParseIP("10.1.1.254"), 0, time.Second, time.Second, "", "", "", true)
	if err != nil {
		t.Fatalf("Could not create session: %s", err)
	}
	defer session1.Close()
```

as the addresses need to be good address.
That code, with the counterpart in the golden file, distract a little from the purpose of the test, that is the loglevel value.

Also, even if `sessionManager.reloadConfig` is a private field, it's already used by tests for closing the channel. I don't know if it's worse than a dumb session.